.gitmodules

```
@@ -1,6 +1,12 @@
 [submodule "libsecutils"]
 	path = libsecutils
-	url = ../libsecutils.git
+	url = https://github.com/siemens/libsecutils.git
 [submodule "cmpossl"]
 	path = cmpossl
-	url = ../../mpeylo/cmpossl.git
+	url = https://github.com/mpeylo/cmpossl.git
+[submodule "LightweightCmpRa"]
+	path = LightweightCmpRa
+	url = git@github.com:siemens/LightweightCmpRa.git
+[submodule "cmp-ra-component"]
+	path = cmp-ra-component
+	url = git@github.com:siemens/cmp-ra-component.git
```

Sorry for jumping in @DDvO, I'm subscribed here. I think submodules with public github repositories (where no credentials are needed) should consistently use HTTPS URLs, it makes life less painful in CI builds.

```suggestion
	url = https://github.com/mpeylo/cmpossl.git
[submodule "LightweightCmpRa"]
	path = LightweightCmpRa
	url = https://github.com/siemens/LightweightCmpRa.git
[submodule "cmp-ra-component"]
	path = cmp-ra-component
	url = https://github.com/siemens/cmp-ra-component.git
```

---

Thanks @nejch for your good hint - fixed.